lib/sidekiq-status/web.rb

```diff
@@ -142,53 +78,74 @@ def has_sort_by?(value)
         ]
 
         @headers.each do |h|
-          h[:url] = "statuses?" + params.merge("sort_by" => h[:id], "sort_dir" => (sort_by == h[:id] && sort_dir == "asc") ? "desc" : "asc").map{|k, v| "#{k}=#{CGI.escape v.to_s}"}.join("&")
+          h[:url] = "statuses?" + qparams(
+            "sort_by" => h[:id],
+            "sort_dir" => (sort_by == h[:id] && sort_dir == "asc") ? "desc" : "asc"
+          )
           h[:class] = "sorted_#{sort_dir}" if sort_by == h[:id]
         end
 
-        erb(sidekiq_status_template(:statuses))
+        erb(:statuses, views: VIEWS)
```

Comment:
The erb method started supporting the view argument in Sidekiq v7.3, so running it on versions below v7.3 will likely cause an error. https://github.com/sidekiq/sidekiq/commit/778de29471b77a089cfce11a0ddca08abc0c0512#diff-0b5e33c5f672a6fd44092604b8c74be6c3283286b9ce0976d8a416736bf0b95eR50-R51